python/triton/testing.py

```diff
@@ -18,15 +18,16 @@ def nvsmi(attrs):
 
 
 def _summarize_statistics(times, quantiles, return_mode):
-    import torch
+    import numpy as np
+    times = np.asarray(times, dtype=np.float32)
     if quantiles is not None:
-        ret = torch.quantile(times, torch.tensor(quantiles, dtype=torch.float)).tolist()
+        ret = np.quantile(times, np.array(quantiles, dtype=np.float32)).tolist()
```

Comment:
Introducing a runtime dependency on `np` is not improving anything. I would probably be fine with staying in python lists and adding a short python implementation of each return mode. The hardest is probably quantile, and even that's fairly simple:
```python
def quantile(times, quantiles):
    times = sorted(times)
    scale = len(times) - 1
    def get_quantile(q):
        point = q * scale
        lower = math.floor(point)
        upper = math.ceil(point)
        t = point - lower
        return (1 - t) * times[lower] + t * times[upper]
    return [get_quantile(q) for q in quantiles]
```
(untested, but you get the idea)

As a bonus, `torch` is massively overhead-bound for small tensors so this probably performs better anyway.